app/controllers/BulkFileUploadController.scala

```diff
@@ -215,8 +216,7 @@ class BulkFileUploadController @Inject()(
     )(S3UploadError.apply)(S3UploadError.unapply)
   )
   private def getBulkEntryDetails(answers: Option[UserAnswers]): Call = answers.flatMap(_ .get(CustomsRegulationTypePage)) match {
-    case Some(CustomsRegulationType.UnionsCustomsCodeRegulation)  => routes.ArticleTypeController.onPageLoad(NormalMode)
-    case Some(CustomsRegulationType.UKCustomsCodeRegulation) => routes.UkRegulationTypeController.onPageLoad(NormalMode)
-    case _ => routes.UkRegulationTypeController.onPageLoad(NormalMode)
+    case Some(CustomsRegulationType.UnionsCustomsCodeRegulation)  => routes.EntryDetailsController.onPageLoad(NormalMode)
```

Comment:
Hi @simms11 seems that this match is doing the same in each case. Is there a better way of writing this?